src/index.js

```diff
@@ -33,7 +33,15 @@ class Signer {
    * @param {Types.SignOptions} options
    * @returns {URL} The signed url.
    */
-  sign({ bucket, key, checksum, expires, sessionToken, publicRead }) {
+  sign({
+    bucket,
+    key,
+    checksum,
+    contentLength,
```

Comment:
```suggestion
    contentLength = 0,
```

Just bit more inline with typedefs